daily-challenges/2024-12/11/ducnq.k11.spec.ts

```
@@ -0,0 +1,24 @@
+        return null;
+    }
+    for(let i = 0; i < menhGia.length; i++){
+        let soLuong = Math.floor(amount/menhGia[i]);
+        if(soLuong > 0){
+            result[menhGia[i]] = soLuong;
+            amount -= soLuong*menhGia[i];
+        }
+    }
+    return result;
+}
+console.log(moneyExchange(1700000));
```

Biome (1.9.4)

[error] Reassigning a function parameter is confusing.

The parameter is declared here:

Use a local variable instead.

(lint/style/noParameterAssign)